Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN ./rebar3 get-deps
 COPY src/ src/
 RUN ./rebar3 escriptize; find . -type f -executable
 
-FROM erlang AS runner
```

Comment:
Please use the same base image across all layers!

This image has been written in a way such that in line 1 an `erlang` base is defined and reused across stages.

Alpine in particular is know to cause problems.

I agree though that an update is required and we should probably go from 22 to 24 or 25, depending on what is currently the most common version across Ubuntu/Debian/RH desktop editions.

Reply:
My plan was to run the tests, and see what would happen, but there were none, so my plan was to close this pr.

This work was btw apart of: https://forum.exercism.org/t/docker-image-sizes/6478

Reply:
Thanks for pointing me towards that thread, I will see what I can do.

I will also experiment with alpine.

Tests should exist though, but the require to have Erlang installed locally. I need to document how to run them, and perhaps also make them part of the CI :D

Reply:
I mean it would also be fine if you don't, the erlang docker containers are rather small. The lfe (lisp-flavored erlang) was 1,5 gb (which is a bit too big), so we found a way to reduce it to like 100 mb.

Reply:
I know LFE (dabbled with it myself).

Knowing how far down I can get for a Elixir Phoenix application using `nix`' `dockerTools` I am still disappointed by 100 MiB…